tests/tests.sql

```diff
@@ -18,130 +36,35 @@ insert into db.people_cris (crash_id, unit_nbr, prsn_nbr, is_primary)
-
--- override cris road type by setting vz to 4 (highway)
-update db.crashes_vz set road_type_id = 4 where crash_id = 1;
-
--- verify that cris crash id 1 has road_type_id 3 (arterial)
-select * from people_cris_view;
--- verify that vz records have road_type_id 4 (highway)
-select * from people_vz_view;
--- verify that unified records have road type 4 (highway)
-select * from people_unified_view;
-
--- revert vz road type edit
-update db.crashes_vz set road_type_id = null where crash_id = 1;
-
--- verify that cris crash id 1 has road_type_id 3 (arterial)
-select * from people_cris_view;
--- verify that vz records have are blank except for IDS
-select * from people_vz_view;
--- verify that unified records have road type 4 (arterial)
-select * from people_unified_view;
-
-
+-- moves location to location_id 
+update db.crashes_cris set latitude = 30.16290408, longitude = -97.65251883 where crash_id = 1;
+-- verify that unified record has location ID 6BDBB50336
+select * from people_unified_view where crash_id = 1;
+-- override cris lat lon
+update db.crashes_cris set latitude = 30.35808625, longitude = -97.68813396 where crash_id = 1;
+-- verify that unified record has location ID 153D618A97
+select * from people_unified_view where crash_id = 1;
+-- verify 1 insert event and two update events in the change log
+select * from db.change_log where record_type = 'crashes' and record_id = 1;
 -------------------------------
--- 2. Update units
+-- 4. Update units
 -------------------------------
 -- update one cris unit to be bicycle 
--- (we are simulating an import here by using crash_id + unit_nbr)
 update db.units_cris set unit_type_id = 5 where crash_id = 1 and unit_nbr = 1;
-
--- verify that cris crash id 1 unit nbr 1 has unit type 5 (bicycle)
-select * from people_cris_view;
--- verify that vz records have are blank except for IDS
-select * from people_vz_view;
--- erify that unified crash id 1 unit nbr 1 has unit type 5 (bicycle)
-select * from people_unified_view;
-
--- override unit value in vz to be motorcycle
--- (we are simulating hasura by using unit.id key to update)
-update db.units_vz set unit_type_id = 3 where id = 1;
-
--- verify that vz unit override is taking effect
-select * from people_cris_view;
-select * from people_vz_view;
-select * from people_unified_view;
-
--- update cris unit value, which should have no effect
-update db.units_cris set unit_type_id = 1 where crash_id = 1 and unit_nbr = 1;
-
--- and verify that unified table still reflects vz edit
-select * from people_cris_view;
-select * from people_vz_view;
-select * from people_unified_view;
-
--- revert vz override
-update db.units_vz set unit_type_id = null where id = 1;
-
--- and verify that unified table now reflects cris values
-select * from people_cris_view;
-select * from people_vz_view;
-select * from people_unified_view;
-
+-- find this update in the change log
+select * from db.change_log where record_type = 'units' order by created_at desc limit 1;
```

Comment:
also when i went to search in the change_log i found myself wanting to filter by where `crash_id = 1` to find this record but instead i had to go into the unit table and figure out what id the unit had and then use that to filter by `record_id`... idk im still feeling partial to keep the easy linkage of `crash_id` thru out all the records... but if we do away with custom ids in the tables then that begs the question -- what would be the `record_id` for `record_types` that have compound pkeys?

Reply:
splitting out the changes table makes sense to me. Having them combined into one table like we do now has created a stumbling block when debugging! I think that we often would find ourselves filtering by `record_type` too if we used a combined table.